box.json

```diff
@@ -63,11 +63,11 @@
         "cfpm":"echo '\".engines/acf2021/WEB-INF/cfusion/bin/cfpm.sh\"' | run",
         "cfpm:install":"echo '\".engines/acf2021/WEB-INF/cfusion/bin/cfpm.sh\" install ${1}' | run",
         "install:2021":"run-script cfpm:install zip,orm,mysql,postgresql,sqlserver,document,feed",
-        "contentbox:install":"install && cd modules/contentbox && install",
-        "contentbox:migrate:create":"migrate create name=${1} migrationsDirectory=modules/contentbox/migrations",
-        "contentbox:migrate":"migrate up migrationsDirectory=modules/contentbox/migrations",
+        "contentbox:install":"install --force && cd modules/contentbox && install --force",
```

Comment:
Note:  I added the `--force` flag here because, as we go forward we will need to make sure the module and framework updates for subsequent releases are up to date.  I think it makes sense, if they are running this script to perform an up-to-date install

Reply:
agreed